modules/rdg/include/bcs/AEFVBC.h

```diff
@@ -0,0 +1,63 @@
+#ifndef AEFVBC_H
+#define AEFVBC_H
+
+#include "IntegratedBC.h"
+#include "BoundaryFluxBase.h"
+
+//Forward Declarations
+class AEFVBC;
+
+template<>
+InputParameters validParams<AEFVBC>();
+
+/**
+ * A boundary condition kernel for the advection equation
```

Comment:
Kernels are volumetric terms, they derive from class called `Kernel`. Boundary conditions derive from `IntegratedBC` or `NodalBC`.  You want to say "A boundary condition object..." here, below, and possibly everywhere else...